INN/umbrella-citylimits
# 54 custom donation block

## Changes

This pull request makes the following changes:

- Adds a donation block
- Removes a 1px-wide transparent border on the right-hand side of `img.footer-skyline.png` that was causing a very thin grey line seen in some of these screenshots

Here's the controls: 

![Screen Shot 2019-11-11 at 13 15 18 ](https://user-images.githubusercontent.com/1754187/68610251-53b8c180-0485-11ea-9118-1fc0b21f7907.png)


Set the headline, CTA text, button text, and button link. I'm using sample text sourced from https://www.damninteresting.com/this-place-is-not-a-place-of-honor/

Here's how it displays at various sizes. Note that the left-aligned version will bump out to the left, to match the mockups. No other blocks have this behavior.

![Screen Shot 2019-11-11 at 12 40 19 ](https://user-images.githubusercontent.com/1754187/68609754-2f101a00-0484-11ea-99a7-0eec6395c602.png)
![Screen Shot 2019-11-11 at 12 40 24 ](https://user-images.githubusercontent.com/1754187/68609755-2fa8b080-0484-11ea-96df-b5e7f2f2bf47.png)
![Screen Shot 2019-11-11 at 12 40 32 ](https://user-images.githubusercontent.com/1754187/68609756-2fa8b080-0484-11ea-91db-a545453e61ac.png)
![Screen Shot 2019-11-11 at 12 40 44 ](https://user-images.githubusercontent.com/1754187/68609757-2fa8b080-0484-11ea-88a6-3e6c5e9b13f0.png)
![Screen Shot 2019-11-11 at 12 40 50 ](https://user-images.githubusercontent.com/1754187/68609759-2fa8b080-0484-11ea-805e-a92b80a87458.png)

This last image shows hover styles for the button, which are the site's default styles for `.btn.btn-primary`.

## Why



For #54

## Testing/Questions

Features that this PR affects:

- places that use blocks


Questions that need to be answered before merging:

- [ ] The center-aligned block looks kinda odd; should we center its text?
- [ ] I've only tested this in the two-column template; do we want to test it on the single-column template?

Steps to test this PR:

1. In a post, add a "City Limits Donation Block" block to the post.
2. Configure its headline, copy, button and link.
3. Update the post
4. View the post
